logstash-output-OMS.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |s|
   s.metadata = { "logstash_plugin" => "true", "logstash_group" => "output" }
 
   # Gem dependencies
-  s.add_runtime_dependency "logstash-core", ">= 2.0.0", "< 3.0.0"
+  s.add_runtime_dependency "logstash-core", ">= 1.4.0", "< 2.0.0"
```

Comment:
this is only for 1.x, as the development of plugins for 2.x changed a bit. I will check if there is a way we can have the same code working for both branches, but AFAIK it is not possible. Maybe we need two branches 

> On 15 Jun 2016, at 18:12, Anurag Gupta <notifications@github.com> wrote:
> 
> In logstash-output-OMS.gemspec:
> 
> > @@ -18,7 +18,7 @@ Gem::Specification.new do |s|
> >    s.metadata = { "logstash_plugin" => "true", "logstash_group" => "output" }
> >  
> >    # Gem dependencies
> > -  s.add_runtime_dependency "logstash-core", ">= 2.0.0", "< 3.0.0"
> > +  s.add_runtime_dependency "logstash-core", ">= 1.4.0", "< 2.0.0"
> Would adding this cause Version 2.3 to not be supported?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>